### PR TITLE
new component AppNavigationCaption

### PR DESCRIPTION
**Pull request by korelstar**
see https://github.com/nextcloud/notes/pull/313#issuecomment-499928900

### CODE REVIEW

File: src/components/AppNavigationCaption/AppNavigationCaption.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+			required: true
+		}
+	}
+}
+</script>
+<style scoped>
+.app-navigation-caption--item {
+	font-weight: bold;
+	color: var(--color-text-maxcontrast);
+	line-height: 44px;
```
Comment (skjnldsv) [inline src/components/AppNavigationCaption/AppNavigationCaption.vue]: Please use the $clickable-area variable :)
https://github.com/nextcloud/nextcloud-vue/blob/c886d5c852ba89580c4ab6dd098a9fe0591197c6/src/assets/variables.scss#L26

```suggestion
	line-height: $clickable-area;
```

File: src/components/AppNavigationCaption/AppNavigationCaption.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	padding: 0 44px;
```
Comment (skjnldsv) [inline src/components/AppNavigationCaption/AppNavigationCaption.vue]:
```suggestion
	padding: 0 $clickable-area;
```

Though the padding seems a bit too big (88px?)

File: src/components/AppNavigationCaption/AppNavigationCaption.vue [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	opacity: 0.7;
+	box-shadow: none !important;
+	pointer-events: none;
+}
+
+.app-navigation-caption--item:not(:first-child) {
+	margin-top: 22px;
```
Comment (skjnldsv) [inline src/components/AppNavigationCaption/AppNavigationCaption.vue]:
```suggestion
	margin-top: $clickable-area / 2;
```

File: src/components/AppNavigationCaption/AppNavigationCaption.vue (hunk 2 above)
Comment (korelstar) [reply src/components/AppNavigationCaption/AppNavigationCaption.vue]: This is how it was in `server`. We could change it though ...

Comment (skjnldsv) [reply src/components/AppNavigationCaption/AppNavigationCaption.vue]: I assumed it was :stuck_out_tongue: 
No idea, it just did :boom: in my head when I read this.
I'm guessing this is for the left padding? To replace the icon?

If so `padding-left` should be enough?

Comment (korelstar) [reply src/components/AppNavigationCaption/AppNavigationCaption.vue]: I will try it.

Comment (korelstar) [reply src/components/AppNavigationCaption/AppNavigationCaption.vue]: All done.

**Comment by korelstar** [comment]
Can this be merged, now @skjnldsv ?

**Comment by ChristophWurst** [comment]
@skjnldsv merge

**Pull request merged**